src/results.cpp

```diff
@@ -370,38 +367,42 @@ util::Optional<Mixed> Results::aggregate(size_t column, bool return_none_for_emp
 
 util::Optional<Mixed> Results::max(size_t column)
 {
-    return aggregate(column, true, "max",
-                     [=](auto const& table) { return table.maximum_int(column); },
-                     [=](auto const& table) { return table.maximum_float(column); },
-                     [=](auto const& table) { return table.maximum_double(column); },
-                     [=](auto const& table) { return table.maximum_timestamp(column); });
+    size_t return_ndx;
+    auto results = aggregate(column, "max",
+            [&](auto const& table) { return table.maximum_int(column, &return_ndx); },
```

Comment:
The indentation of these lambdas is a bit wonky. The indentation that was there previously made more sense to me.